Buildings/Fluid/HeatExchangers/DXCoils/AirCooled/Examples/BaseClasses/AirsideEconomizerController.mo

```diff
@@ -0,0 +1,94 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.AirCooled.Examples.BaseClasses;
+model AirsideEconomizerController "Airside economizer controller"
+
+  parameter Real gai(min=Modelica.Constants.small) = 1
+    "Proportional gain of controller"
+    annotation(Dialog(group="Control"));
+  parameter Modelica.SIunits.Time Ti
+    "Integrator time"
+    annotation(Dialog(group="Control"));
+  parameter Real minOAFra(min=0,max=1, final unit="1")
+    "Minimum outdoor air fraction";
+  Buildings.Controls.Continuous.LimPID con(
+    Td=1,
+    reverseAction=true,
+    k=gai,
+    yMin=minOAFra,
+    final controllerType=Modelica.Blocks.Types.SimpleController.PI,
+    Ti=Ti) "PID controller"
+    annotation (Placement(transformation(extent={{-80,70},{-60,90}})));
+  Modelica.Blocks.Interfaces.RealInput MATSet(
+    final unit="K",
+    final quantity="ThermodynamicTemperature",
+    displayUnit="degC")
+    "Mixed air setpoint temperature"
+    annotation (Placement(transformation(rotation=0,
+      extent={{-140,60},{-100,100}})));
+  Modelica.Blocks.Interfaces.RealInput MAT(
+    final unit="K",
+    final quantity="ThermodynamicTemperature",
+    displayUnit="degC")
+    "Measured mixed air temperature"
+    annotation (Placement(transformation(rotation=0, extent={{-140,20},
+      {-100,60}})));
+  Modelica.Blocks.Interfaces.RealInput cooMod(
+    final unit="1")
+    "Cooling mode of the cooling system"
+    annotation (Placement(
+        transformation(rotation=0, extent={{-140,-20},{-100,20}})));
+  Modelica.Blocks.Math.RealToBoolean ASEOff(final threshold=1.5)
+    "Determine if airside economizer is off"
+    annotation (Placement(transformation(extent={{-80,-10},{-60,10}})));
+  Modelica.Blocks.Logical.Not not1 "Inverse signal"
+    annotation (Placement(transformation(extent={{-40,-10},{-20,10}})));
+  Modelica.Blocks.Logical.Switch switch1 "Switch to select control output"
+    annotation (Placement(transformation(extent={{40,-10},{60,10}})));
+  Modelica.Blocks.Sources.Constant const(final k=0)
+    "Constant output signal with value 1"
+    annotation (Placement(transformation(extent={{-80,-60},{-60,-40}})));
+  Modelica.Blocks.Interfaces.RealOutput y "Connector of Real output signal"
```

Comment:
@YangyangFu Need to make sure `Input` and `Output` are in the second section. Following the text order: parameter -- interfaces -- blocks/instances -- (protected sections)